blockchain/ethereum.go

```diff
@@ -716,18 +717,23 @@ func (e *EthereumMultinodeClient) WaitForEvents() error {
 }
 
 // SimulatedEthereumURLs returns the websocket URLs for a simulated geth network
-func SimulatedEthereumURLs(e *environment.Environment) ([]*url.URL, error) {
-	return e.Charts.Connections("geth").LocalURLsByPort("ws-rpc", environment.WS)
+func SimulatedEthereumURLs(e *environment.Environment) ([]string, error) {
```

Comment:
Can we combine `SimulatedEthereumURLs` and `SimulatedSoakEthereumURLs` to just `SimulatedEthereumURLs` that first looks for a "local" forwarded port, and if not there, will try and find a "remote" one?

Reply:
Nice catch, fixed, moved paths up